http/css/style.css

```diff
@@ -20,12 +20,17 @@
 body {
     font-size: 85%;
     font-family: sans-serif;
-    background-color: #ffffff;
     display: flex;
     justify-content: center;
+    background: url("../images/background.png") left / cover no-repeat fixed;
```

Comment:
Try to change that by this:

    background-color: #0A3A48;

I find it slicker :smile: